### ISSUE TITLE
Geoserver extension cannot create vector data source from PostgreSQL v10.x repo

### ISSUE THREAD

**Issue by gjsa73** [locationtech/geogig#413]
Extension Version:  geoserver-2.12-SNAPSHOT-geogig-plugin.zip
PostgresSQL version:  10.1

Error when creating new vector data source:
![image](https://user-images.githubusercontent.com/21994806/35253472-b70d45c8-0039-11e8-8c43-2767ed8e7416.png)

**Comment by emerkle826** [comment]
@gjsa73 
I've created PR to fix this issue here: #414 
However, there are no immediate plans to release a new official version of GeoGig that will include this fix. When this is merged into `master` (i.e. 1.3.x), we can port it back to `1.2.x`. Once that is done, you can get the fix by building the `geogig-postgres` module and replacing `geogig-postgres-1.2.0.jar` in your GeoServer install.

**Comment by emerkle826** [comment]
@gjsa73 The fix has been merged. If you can pull the `1.2.x` branch and build the geogig-postgres-1.2-SNAPSHOT.jar from it, you can replace the jar in your GeoServer install with what you build and it should resolve the issue.

**Comment by gjsa73** [comment]
Thanks, finally got back to this and it does indeed resolve it.
